Source/WebGPU/WebGPU/USDModel.swift

```diff
@@ -1237,7 +1405,7 @@ final class USDModelLoader: _Proto_UsdStageSession_v1.Delegate {
     func textureUpdated(data: consuming sending _Proto_TextureDataUpdate_v1) {
         let identifier = data.identifier
         self.dispatchSerialQueue.async {
-            self.objcLoader.updateTexture(webRequest: Converter.webUpdateTextureRequestFromUpdateTextureRequest(data))
+            self.objcLoader.updateTexture(webRequest: webUpdateTextureRequestFromUpdateTextureRequest(data))
```

Comment:
this file should be split up probably, it’s starting to get pretty big and multiple responsibilities